routes/compat/wakatime/v1/statusbar.go

```diff
@@ -0,0 +1,106 @@
+package v1
+
+import (
+	"net/http"
+	"time"
+
+	"github.com/gorilla/mux"
+	conf "github.com/muety/wakapi/config"
+	"github.com/muety/wakapi/middlewares"
+	"github.com/muety/wakapi/models"
+	v1 "github.com/muety/wakapi/models/compat/wakatime/v1"
+	routeutils "github.com/muety/wakapi/routes/utils"
+	"github.com/muety/wakapi/services"
+	"github.com/muety/wakapi/utils"
+)
+
+type StatusBarViewModel struct {
+	CachedAt time.Time        `json:"cached_at"`
+	Data     v1.SummariesData `json:"data"`
+}
+
+type StatusBarHandler struct {
+	config      *conf.Config
+	userSrvc    services.IUserService
+	summarySrvc services.ISummaryService
+}
+
+func NewStatusBarHandler(userService services.IUserService, summaryService services.ISummaryService) *StatusBarHandler {
+	return &StatusBarHandler{
+		userSrvc:    userService,
+		summarySrvc: summaryService,
+		config:      conf.Get(),
+	}
+}
+
+func (h *StatusBarHandler) RegisterRoutes(router *mux.Router) {
+	r := router.PathPrefix("").Subrouter()
+
+	r.Use(
+		middlewares.NewAuthenticateMiddleware(h.userSrvc).Handler,
+	)
+	r.Path("/users/{user}/statusbar/{range}").Methods(http.MethodGet).HandlerFunc(h.Get)
+	r.Path("/v1/users/{user}/statusbar/{range}").Methods(http.MethodGet).HandlerFunc(h.Get)
```

Comment:
For the sake of consistency, please also add a "compat" route, analogously to this one: https://github.com/muety/wakapi/blob/2f9cafc88c10b5b1384997232418c30d82f04281/routes/compat/wakatime/v1/stats.go#L35